### ISSUE TITLE
failed to run command - you've introduced a cycle in your pipeline that involves the following stages:- job_dir.dvc

### ISSUE THREAD

**Issue by AnsujJoshi** [iterative/dvc#2224]
dvc version = 0.41.3
Platform = DEB(Linux)
Installation method = pip

I am trying to run the command 
```
dvc run -d gs://$INPUT_BUCKET/testing \
        -o gs://$OUTPUT_BUCKET/$JOB_DIR  \
        bash automate.sh
```
where automate.sh runs a dataflow pipeline and ML engine(AI platform) job and return the checkpoints to `gs://$OUTPUT_BUCKET/$JOB_DIR`. It is giving me the following error: 
```
ERROR: failed to run command - you've introduced a cycle in your pipeline that involves the following stages:
	- job_dir.dvc
```
Please help me solve this. Thanks is advance.

**Comment by efiop** [comment]
Hi @AnsujJoshi ! Could you please try the updated dvc version(0.50.1)? We've fixed a few bugs related to that since 0.41.3.

**Comment by AnsujJoshi** [comment]
Thanks it worked.
